### PR TITLE
Add Tor configuration

### PR DESCRIPTION
**Pull request by bludnic**
### Checklist

- [x] Add Tor nodes
- [x] Add `explorer` URL for Tor
- [x] Add infoService URL (for Tor as well)
- [x] Add `nodeMinVersion`
- [x] Add `website` for Tor
- [x] Add `nodeMinVersion`
- [x] Update Swagger types
- [x] Update `README.md`. Add Tor examples.
- [x] Add coin tor nodes
- [x] ⚠️ Renamed `serviceNodes` to `services` (see [lisk config](https://github.com/Adamant-im/adamant-wallets/pull/47/files#diff-2609a459311c596b7ed69b20c3be692fe46d712c1cafa287a0294f35957ad71fR34-R39) @StanislavDevIOS)
 
Use https://editor.swagger.io/ if you want to test the Swagger schema

### CODE REVIEW

**Comment by bludnic** [comment]
[Include tor nodes to adamant-wallets](https://trello.com/c/BIKZJ8ta/383-include-tor-nodes-to-adamant-wallets)

File: README.md [hunk 1]
```diff
@@ -79,6 +81,17 @@ Coin/token info stored in `\general\${token_name}` folders. Specific blockchain
     }
   ],
 
+  // Optional. Services used by the project
```
Comment (adamant-al) [inline README.md:84]:
```suggestion
  // Optional. Services related to a project
```

File: README.md [hunk 2]
```diff
@@ -90,6 +103,19 @@ Coin/token info stored in `\general\${token_name}` folders. Specific blockchain
       "url": "https://example.com/whitepaper.pdf"
     }
   ]
+
+  // Optional. Tor configuration if the project uses Tor
```
Comment (adamant-al) [inline README.md:107]:
```suggestion
  // Optional. Tor configuration if a project uses Tor
```

**Review by adamant-al — CHANGES_REQUESTED** [review]
We forger about coin tor nodes, add them.

**Review by just-software-dev — APPROVED** [review]

**Review by just-software-dev — APPROVED** [review]

**Review by adamant-al — APPROVED** [review]
- Suggest to set `minNodeVersion` to at least `0.7.0`.
- We suppose infoService's `alt_ip`'s default port is 80.
- Consider to add btcnode2 + alt_ip + tor, it's working
- Consider to add dogenode2 + alt_ip + tor, it's working
- Consider to add ethnode2 + alt_ip + tor, it's working
- Dash's node2 and node3 are on the way soon @confleux

**Comment by adamant-al** [comment]
We can merge it.

**Pull request merged**